lib/config.php

```diff
@@ -146,7 +146,10 @@ public function filter_servers() {
         foreach ($this->config as $key => $vals) {
             if (in_array($key, $excluded, true)) {
                 foreach ($vals as $index => $server) {
-                    if (!array_key_exists('server', $server)) {
+                    if (!empty($server['default'])) {
```

Comment:
This returns the error we wanted to fix. The goal of removing this code was to allow you to have the same unique identifier for the default server each time you reconnect. This resulted in losing the configured folders. With this we will have the same problem.